src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala

```diff
@@ -29,7 +29,7 @@ import scala.tools.nsc.util.{ClassPath, ClassRepresentation, EfficientClassPath}
  */
 case class AggregateClassPath(aggregates: Seq[ClassPath]) extends ClassPath {
   override def findClassFile(className: String): Option[AbstractFile] = {
-    val (pkg, simpleClassName) = PackageNameUtils.separatePkgAndClassNames(className)
+    val (pkg, _) = PackageNameUtils.separatePkgAndClassNames(className)
```

Comment:
FWIW, unused warning is suppressed by `simpleClassName @ _` which is lexically equivalent but visually signals that the name is introduced only to document what value is unused. Probably none of these examples are obscure; here, the RHS says "pkg and class name". Could also `rhs._1`.

Reply:
Do you think this would be better?
```
val pkg = PackageNameUtils.separatePkgAndClassNames(className)._1
```